main.py

```diff
@@ -10,6 +11,12 @@
 from compressors.zstd_compressor import ZstdCompressor
 from data import load_20news, load_ohsumed_single_23, load_reuters, load_kinnews_kirnews
 
+def write_csv(filename, data, headers):
```

Comment:
should this be moved to another file? 

(I don't do much python so I don't have a strong opinion)

Reply:
It probably should (weird to have straggly function definition in main). What other file would be best to move it to?

Reply:
let's keep it in the main now that there's a tight coupling between the csv write format and the table print format.
the main is <150 lines which is reasonable enough